docs/restore-individual-tables.md

```diff
@@ -1,76 +1,108 @@
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/tmp/table \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
 ```
 
-The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the `.cfp` file, which contains the transfer key and an encrypted tablespace key.
-
-The files are located in the target directory:
+After these commands finish successfully, you'll need these files for the table import.
 
 ```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
+/data/backups/mysql/test/export_test.cfp
 ```
 
 ## Import the table
 
-Import the table by doing the following:
+Now that we've exported the table, we can move it to the destination system. You need to follow these steps in order on the target server:
+
+* Use the saved schema definition to create an empty table at the destination. This provides the framework for your imported data.
 
-* Create a copy of the table
-* Discard the table's original tablespace
-* Copy the `.ibd` and `.cfg` files to the location
-* Import the tablespace
+* Copy all three important files (.ibd, .cfp, and .cfg) from your source directory to the matching directory in the destination system.
 
-To import that table, do the following steps:
 
-1. On the destination server, running Percona Server for MySQL with XtraDB or MySQL 8.0, create a table with the same structure.
+The following is the detailed process:
 
-2. Run the `ALTER TABLE ... DISCARD TABLESPACE` command to discard the new table's tablespace:
+1. On your target server (which must be running Percona Server for MySQL with XtraDB or MySQL 8.0), use the schema and create an empty copy of the table in the new location.
+
+    ```{.bash data-prompt="$"}
+    $ mysql -uroot < ./schema_dumps/test/create_database_test.sql;
+    $ mysql -uroot < ./schema_dumps/test/export_test.sql;
+    ```
+
+2. Run this SQL command to disconnect the new table from its default tablespace. This prepares it to receive the imported data:
 
     ```{.bash data-prompt="mysql>"}
     mysql> ALTER TABLE test.export_test DISCARD TABLESPACE;
     ```
 
-    If you see the an error message like "ERROR 1809 (HY000): Table 'test/export_test' in system tablespace", enable the `innodb_file_per_table` option on the server and do step 1 and 2 again.
+    If you see an error like "ERROR 1809 (HY000): Table 'test/export_test' in system tablespace", it means the `innodb_file_per_table` setting is disabled on your target server. Enable this setting in your server setup, then try steps 1 and 2 again.
 
-3. Copy the exported files to the `test/` subdirectory of the destination server’s data directory.
+3. Copy all the table files (export_test.ibd, export_test.cfp, and export_test.cfg) from the source /source/backup directory to the matching destination schema directory inside server data directory.
```

Comment:
there is a misunderstanting :joy: I meant you can either write source directory or backup directory